### PR TITLE
feat(perf-issues): Add `View Full Event` button to Span Evidence section

### PR DESCRIPTION
**Pull request by 0Calories**
To make the navigability of Performance Issues less frustrating, we're adding this button to the Span Evidence section, so users can click to view the full event details page.

image

### CODE REVIEW

**Review by DominikB2014 — COMMENTED** [review]
The `KeyValueListDataItem` has an action button prop, I think it makes sense to use that along with the ActionButton component rather then creating a new TransactionRowContainer Component.

You can get similar results, but reusing existing stuff, I played around with this before.
image

**Comment by 0Calories** [comment]
Great suggestion @DominikB2014! I refactored to use the `actionButton` prop and also added some test coverage

File: static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx [hunk 1]
```diff
@@ -257,24 +282,40 @@ const PresortedKeyValueList = ({data}: {data: KeyValueListData}) => (
   <KeyValueList shouldSort={false} data={data} />
 );
 
-const makeTransactionNameRow = (event: Event, orgSlug: string) => {
-  const to = transactionSummaryRouteWithQuery({
+const makeTransactionNameRow = (event: Event, orgSlug: string, projectSlug?: string) => {
```
Comment (DominikB2014) [inline static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx:285]: The reason why this is optional, is just to support the span evidence preview?

**Review by DominikB2014 — APPROVED** [review]

File: static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx (hunk 1 above)
Comment (0Calories) [reply static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx:285]: Yep exactly, it's optional in the preview so we can't guarantee it is always present

File: static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx [hunk 2]
```diff
@@ -257,24 +282,40 @@ const PresortedKeyValueList = ({data}: {data: KeyValueListData}) => (
...
+    id: event.eventID,
+    project: projectSlug,
+  });
+
+  const eventDetailsLocation = getTransactionDetailsUrl(orgSlug, eventSlug);
+
+  const actionButton = projectSlug ? (
+    <Button size="xs" to={eventDetailsLocation}>
+      {t('View Full Event')}
+    </Button>
```
[the remark this answers is not in the document]
Comment (0Calories) [reply static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.tsx:303]: lol yeah I think I'm gonna leave it, it doesn't really make a difference

**Pull request merged**